### PR TITLE
Create root-level OWNERS file

### PR DESCRIPTION
**Pull request by fejta**
/assign @erain 

After this `/approve` should work as expected.

Do we want to enable blunderbuss to have prow automatically assign a reviewer on new PRs?

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# See Config struct in https://github.com/kubernetes/test-infra/blob/master/prow/repoowners/repoowners.go
+
+reviewers:
```
Comment (erain) [inline OWNERS]: Could you add both Nick and Xin as reviewers as well?
```
+- nlopezgi
+- xingao267
```

Comment (nlopezgi) [reply OWNERS]: lets just add nlopezgi (we'll add xingao267 later)

Comment (fejta) [reply OWNERS]: Done

**Comment by fejta** [comment]
The travis CI failure is expected because my PR uses my fork

**Pull request merged**